creature_spawner/scripts/mods/creature_spawner/creature_spawner.lua

```diff
@@ -559,6 +562,24 @@ mod.assist_player = function(self)
   end
 end
 
+mod.toggle_rapid_cooldown = function ()
```

Comment:
Is there a way to accomplish this with a regular hook on wherever the cooldown is checked? I'd rather not use a hook_require if possible

Reply:
Hm, the whole function is kind of a crutch tbh (resetting **all** abilities to their default). 

I'll ask around on Discord if there is a better way to get current ability's cooldown..